autoscale-core/src/main/java/com/hpe/caf/autoscale/core/ScalerThread.java

```diff
@@ -217,4 +249,9 @@ private int establishMemLimitReached(final double currentMemoryLoad)
         }
         return 0;
     }
+
+    private boolean timelimitExceeded()
+    {
+        return (System.currentTimeMillis() - firstAttempt) >= (2 * 60 * 1000);
```

Comment:
I'm not sure what this 2 minutes is for but should it be configurable?

Reply:
This is the time that the governor will spend attempting to make room for the service before giving up